client-src/elements/chromedash-guide-metadata-page.ts

```diff
@@ -230,7 +233,7 @@ export class ChromedashGuideMetadataPage extends LitElement {
 
   render() {
     return html`
-      ${this.renderSubheader()}
+      ${this.loading ? nothing : this.renderSubheader()}
```

Comment:
This line should be reverted so that it always calls `renderSubheader()`.

Reply:
Ah, sorry about that - I've removed these changes.